tests/test_core.py

```diff
@@ -668,6 +668,22 @@ def test_interpolate(test_df_year):
     assert not test_df_year.filter().data.duplicated().any()
 
 
+def test_interpolate_datetimes(test_df):
+    # test that interpolation also works with date-times.
+    some_date = datetime.datetime(2007, 7, 1)
+    if test_df.time_col == "year":
+        with pytest.raises(ValueError):
+            test_df.interpolate(some_date)
+    else:
+        test_df.interpolate(some_date)
+        obs = test_df.filter(time=some_date).data['value']\
+            .reset_index(drop=True)
+        exp = pd.Series([3, 1.5, 4], name='value')
+        pd.testing.assert_series_equal(obs, exp, check_less_precise=True)
+        test_df.interpolate(some_date)
```

Comment:
can you please also insert the comment from the test above (`# redo the interpolation and check that no duplicates are added`) to make it easier for the next person looking at this test to know why this is done...